### ISSUE TITLE
Problem in reproducing results for CCC-High

### ISSUE THREAD

**Issue by blaze1522** [oripress/CCC#4]
Dear authors,

I was trying to reproduce the results in Table 1 of the paper. For pretrained, the mean accuracy is 1.5%, and for batchnorm it is 6.8% however, I am getting the following results.  

```
Mean Accuracy on model_pretrained_baseline_10.0_transition+speed_5000_seed_43.txt : 7.936545185026257
Mean Accuracy on model_pretrained_baseline_10.0_transition+speed_2000_seed_43.txt : 7.936228248741557
Mean Accuracy on model_pretrained_baseline_10.0_transition+speed_1000_seed_44.txt : 7.919677122934722

Mean Accuracy on model_batchnorm_baseline_10.0_transition+speed_1000_seed_44.txt : 21.110168977564744
Mean Accuracy on model_batchnorm_baseline_10.0_transition+speed_1000_seed_43.txt : 20.893276569639017
```

Please guide me towards my mistake. Please do ask for any details that you need.

**Comment by oripress** [comment]
Hi, 
CCC-Hard is generated with a baseline accuracy of 0, not 10. 
Does this solve the problem?

**Comment by blaze1522** [comment]
Okay, I'll generate again and get back on this.

Also, to confirm. CCC-Easy is generated with a baseline of 40, and CCC-Medium is generated with a baseline of 20, right?

**Comment by oripress** [comment]
Yes, correct. I'll make that clearer in the readme.

**Comment by blaze1522** [comment]
It's working. Thanks for the help!
